### PR TITLE
synopkg: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/synopkg.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# synopkg
+
+> Package management utility for Synology DiskStation Manager.
+> More information: <https://www.synology.com/dsm>.
+
+- List installed packages without description:
```
Comment (CleanMachine1) [inline pages/linux/synopkg.md:6]:
```suggestion
- List installed packages without a description:
```

File: pages/linux/synopkg.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`synopkg list --name`
+
+- List packages depend on a specific package:
```
Comment (CleanMachine1) [inline pages/linux/synopkg.md:10]:
```suggestion
- List packages which depend on a specific package:
```

File: pages/linux/synopkg.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Print the status of a package:
+
+`synopkg status {{package}}`
+
+- Uninstall a package:
+
+`sudo synopkg uninstall {{package}}`
+
+- Check if a package is updateable:
```
Comment (CleanMachine1) [inline pages/linux/synopkg.md:26]:
```suggestion
- Check if a package is updatable:
```

File: pages/linux/synopkg.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`synopkg checkupdate {{package}}`
+
+- Upgrade all packages to latest version:
+
+`sudo synopkg upgradeall`
+
+- Install a package from file:
```
Comment (CleanMachine1) [inline pages/linux/synopkg.md:34]:
```suggestion
- Install a package from a `synopkg` file:
```
```suggestion
- Install a package from a synopkg file:
```

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/synopkg.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- List installed packages without a description:
+
+`synopkg list --name`
+
+- List packages which depend on a specific package:
+
+`synopkg list --depend-on {{package}}`
+
+- Start/Stop a package:
```
Comment (marchersimon) [inline pages/linux/synopkg.md:14]: Not entirely sure about the capitalization here, but I think it should be like this:
```suggestion
- Start/stop a package:
```
Even tough I'd actually prefer
```suggestion
- Start or stop a package:
```

File: pages/linux/synopkg.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Uninstall a package:
+
+`sudo synopkg uninstall {{package}}`
+
+- Check if a package is updatable:
+
+`synopkg checkupdate {{package}}`
+
+- Upgrade all packages to latest version:
```
Comment (marchersimon) [inline pages/linux/synopkg.md:30]:
```suggestion
- Upgrade all packages to the latest version:
```

File: pages/linux/synopkg.md (hunk 5 above)
Comment (navarroaxel) [reply pages/linux/synopkg.md:14]: I copied the format used in the `systemctl` page.

Comment (marchersimon) [reply pages/linux/synopkg.md:14]: I wouldn't consider many docs trustworth in terms of correct grammer. @SethFalco what would you say?

Comment (SethFalco) [reply pages/linux/synopkg.md:14]: > I wouldn't consider many docs trustworth in terms of correct grammer.

Agreed! It's always handy to improve docs. Most devs aren't very passionate about technical writing. ^-^'

---

I don't know which is valid between "Start/Stop" vs "Start/stop" to be honest. 🤔
I can't say I specifically have a preference between the 2 suggestions, either.

Probably depends on what the repository prefers:
* Something technical users would be comfortable consuming.
* Formal writing that could be in articles, books, education, etc.

Both are unambiguous, easy to consume, and familiar to users. The top is _probably_ more common for technical readers, while the bottom is negligibly more verbose but more formal, i.e. for articles or education.

TL;DR: I can't provide a strong opinion, but maybe that'd help?

File: pages/linux/synopkg.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+- Print the status of a package:
+
+`synopkg status {{package}}`
+
+- Uninstall a package:
+
+`sudo synopkg uninstall {{package}}`
+
+- Check if a package is updatable:
```
[the remark this answers is not in the document]
Comment (marchersimon) [reply pages/linux/synopkg.md:26]: I like the second one

[the remark this answers is not in the document]
Comment (CleanMachine1) [reply pages/linux/synopkg.md:26]: I like the first one.

File: pages/linux/synopkg.md (hunk 4 above)
Comment (sbrl) [reply pages/linux/synopkg.md:34]: The second suggestion from @CleanMachine1 works best I think. The enclosing backticks "`" are not necessary in this instance I don't think.

File: pages/linux/synopkg.md (hunk 5 above)
Comment (sbrl) [reply pages/linux/synopkg.md:14]: I suggest `Start or stop a package:` would be the best option here. If you really want the forward slash, then `Start/Stop` would be my preferred suggestion.

File: pages/linux/synopkg.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (sbrl) [reply pages/linux/synopkg.md:26]: The second one is the most clear I think for me.

**Review by marchersimon — APPROVED** [review]
My suggestion above is still unresolved, but I'm fine with both solutions.
Thanks @navarroaxel

**Comment by navarroaxel** [comment]
@marchersimon but Sbrl says the current text is fine.

**Comment by marchersimon** [comment]
My fault for poorly expressing myself: I meant I'm fine with my suggestions or the way it is now.
